.babelrc.js

```diff
@@ -8,9 +8,9 @@ const babelEnv = params => [
 module.exports = api => {
   api.cache(true);
   return {
-    presets: [babelEnv({modules: false}), '@babel/preset-react'],
+    presets: [babelEnv({modules: false}), '@babel/preset-react', '@babel/preset-flow'],
     plugins: [
-      '@babel/plugin-transform-flow-strip-types',
```

Comment:
can this affect our app current build process?

Reply:
I put flow preset there instead, I think it does the same thing

Reply:
aw right, didn't connect the dots immediately, sorry 👍